### DISCUSSION TITLE
Where is the Wiki

### DISCUSSION THREAD

**Discussion by aubreyz** [iSoron/uhabits#1538] [Feature Requests]
Greetings all users of this fantastic gift. 

Various past issues and discussions refer to the Wiki - for example on the mechanics of tasker integration. However the Wiki links do not seem to go anywhere and there is no pointers to a Wiki on the main readme. Where are these useful notes now, and in particular Tasker integration which was on the Wiki?

**Comment by iSoron** [comment]
Hi @aubreyz, the wiki was removed. Most information has been migrated elsewhere, with the notable exception of the Tasker guide you mention. Here is an archived copy:

https://web.archive.org/web/20201103060153/https://github.com/iSoron/uhabits/wiki/Integration-with-Tasker
